homeassistant/components/mealie/__init__.py

```diff
@@ -55,7 +55,11 @@ async def async_setup_entry(hass: HomeAssistant, entry: MealieConfigEntry) -> bo
     except MealieConnectionError as error:
         raise ConfigEntryNotReady(error) from error
 
-    if not version.valid or version < MIN_REQUIRED_MEALIE_VERSION:
+    if not version.valid:
+        LOGGER.warning(
+            "It seems like you are using the nightly version of Mealie, nightly versions could have changes that stop this integration working"
```

Comment:
Please break long string around max 88 characters per line.

Reply:
Thanks for the heads up, rulers now set in vs code and PR created
https://github.com/home-assistant/core/pull/122635